lib/upgradedb/mysql.2020081900.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * LMS version 1.11-git
+ *
+ *  (C) Copyright 2001-2020 LMS Developers
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License Version 2 as
+ *  published by the Free Software Foundation.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307,
+ *  USA.
+ *
+ */
+
+$this->BeginTrans();
+
+$this->Execute("DROP VIEW IF EXISTS vusersadmin");
```

Comment:
Dobrze by było w widokach nie robić `vusersadmin`, a po prostu `vuseradmin` - już kiedyś mieliśmy widoki z `s` i przeszliśmy na bez `s`. Chodzi o występowanie w środku. Jeśli już to może `vuseradmins`?

Reply:
Jak by dać `vallusers` to byłoby wiadomo po nazwie, że zwraca od razu wszystkich.

Reply:
Poprawiłem na 'vallusers'.